apisix/plugins/prometheus/exporter.lua

```diff
@@ -352,13 +362,47 @@ local function etcd_modify_index()
 end
 
 
+local shared_dicts
+local function shared_dict_status()
+    local config_indexs = {"http", "stream", "meta"}
+    if not shared_dicts then
+        shared_dicts = {}
+        local existed = {}
```

Comment:
what `existed` is used for? It looks like `shared_dicts`  will only be `nil` or has `shared_dict_name`.

Reply:
> what `existed` is used for? It looks like `shared_dicts` will only be `nil` or has `shared_dict_name`.

I found that we will configure duplicate `lua_shared_dict` in https://github.com/apache/apisix/blob/master/conf/config-default.yaml, like `prometheus-metrics`, `existed` is used to deduplicate.